rules/camunda-cloud/escalation-boundary-event-attached-to-ref.js

```diff
@@ -0,0 +1,48 @@
+const {
+  is,
+  isAny
+} = require('bpmnlint-utils');
+
+const {
+  getEventDefinition,
+  isAnyExactly
+} = require('../utils/element');
+
+const { ERROR_TYPES } = require('../utils/error-types');
+
+const { reportErrors } = require('../utils/reporter');
+
+const { skipInNonExecutableProcess } = require('../utils/rule');
+
+module.exports = skipInNonExecutableProcess(function() {
+  function check(node, reporter) {
+    if (!isAny(node, [ 'bpmn:CatchEvent', 'bpmn:ThrowEvent' ])) {
```

Comment:
Only boundary events matter here:

```suggestion
    if (!is(node, 'bpmn:BoundaryEvent'))
```

Reply:
Ooops! Good catch! 🖐🏻 ⚾